apps/infrastructure/src/modules/chatbot/dynamodb.tf

```diff
@@ -0,0 +1,28 @@
+# Chatbot queries
+
+module "dynamodb_chatbot_queries" {
+  source = "git::github.com/terraform-aws-modules/terraform-aws-dynamodb-table.git?ref=715399dbe24f6443820bf5de80f6100b35d56355" # v4.0.0
+
+  billing_mode                = "PAY_PER_REQUEST"
+  deletion_protection_enabled = true
+
+  name                           = "ChatbotQueries"
+  hash_key                       = "id"
+  range_key                      = "createdAt"
+  server_side_encryption_enabled = true
+
+  attributes = [
+    {
+      name = "id"
+      type = "S"
+    },
+    {
+      name = "createdAt"
+      type = "S"
+    },
+    {
+      name = "data"
+      type = "S"
+    },
```

Comment:
The terraform resource forces to only define attributes that make part of the primary index (hash + sort key) or global/local secondary indices ([source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table#dynamodb-table-attributes))

At this point, I was thinking whether it would be feasible to extract the content of the data map into single fields.
This way, we can select the ones that will be actually indexed and include them in this list of attributes afterwards

This, of course, after having decided what kind of queries and indices the application needs (see previous comment)